package.json

```diff
@@ -10,11 +10,11 @@
     "tdd": "cross-env BABEL_ENV=test NODE_ENV=unittest jest --watch",
     "lint": "yarn run lint-prettier && eslint --cache --cache-location '.eslintcache/' --ext .js --ext .jsx src",
     "lint-prettier": "node prettier.js lint",
-    "start": "razzle start",
-    "start-without-ssr": "cross-env RAZZLE_DISABLE_SSR=true razzle start",
-    "start-with-local-graphql": "cross-env LOCAL_GRAPHQL_API=true razzle start",
-    "start-with-local-graphql-and-article-converter": "cross-env LOCAL_GRAPHQL_API=true RAZZLE_LOCAL_ARTICLE_CONVERTER=true razzle start",
-    "start-with-local-article-converter": "cross-env RAZZLE_LOCAL_ARTICLE_CONVERTER=true razzle start",
+    "start": "cross-env NDLA_ENVIRONMENT=dev razzle start",
```

Comment:
Change `const ndlaEnvironment = getEnvironmentVariabel('NDLA_ENVIRONMENT', 'test');` to `const ndlaEnvironment = getEnvironmentVariabel('NDLA_ENVIRONMENT', 'dev');` so that we can still override NDLA_ENVIRONMENT.